runme.sh

```diff
@@ -139,7 +139,7 @@ export ARCH=arm
 export GCC5_AARCH64_PREFIX=$CROSS_COMPILE
 export WORKSPACE=$ROOTDIR/build/tianocore
 export PACKAGES_PATH=$WORKSPACE/edk2:$WORKSPACE/edk2-platforms:$WORKSPACE/edk2-non-osi
-source  edk2/edksetup.sh
+PYTHON_COMMAND=/usr/bin/python3 source  edk2/edksetup.sh
```

Comment:
Actually, it does not work with Python 3.9. I've reported to EDk2. Python 3.8 works.